client/hotrod-client/src/main/java/org/infinispan/client/hotrod/logging/Log.java

```diff
@@ -399,4 +399,8 @@ public interface Log extends BasicLogger {
    @Message(value = "OpenTelemetry API is present in the classpath, but the tracing propagation is not enabled. Client context tracing will not be propagated.", id = 4111)
    void openTelemetryPropagationDisabled();
```

Comment:
This and openTelemetryPropagationEnabled may want to be TRACE since they are logged for every cache that is used when telemetry is on the classpath.  DEBUG may be fine, just want to make sure this spot is considered for evaluation.